docs/reference/export_formats.md

```diff
@@ -0,0 +1,149 @@
+# InvenioRDM records export formats
+
+_Introduced in v12.0.0b1_
+
+InvenioRDM provides various record serialisation formats to allow users to easily export bibliographic records in a machine-readable way or transfer them to different systems. These formats adhere to widely used metadata standards such as DataCite, Dublin Core, BibTeX, MARCXML, DCAT-AP or GeoJSON, which is commonly used for geospatial metadata.
+
+By providing a range of export formats, Invenio-app-rdm enables users to share and exchange metadata records with other systems in a format that is compatible with their respective standards. This makes it easier to ensure that metadata records are accurate, complete, and consistent across different systems.
+
+A bibliographic record can be exported to a different format in a couple of ways:
+1. in the UI, using the `Export` dropdown menu in the record landing page (screenshot with the menu open)
+2. via REST APIs, using the `Accept` header:
+
+
+```
+GET /api/records/ HTTP/1.1
+Host: inveniordm.web.cern.ch
+Accept: <export-format-mime-type>;charset=UTF-8
+```
+
+Find below the `Accept` header MIME type for each export format.
+
+## JSON
+
+The JSON export format in InvenioRDM exports metadata records in a structured JSON format, which includes all the metadata fields associated with the record, as well as any custom fields that have been defined by the community. This makes it possible to export metadata records from Invenio-app-rdm in a machine-readable format that can be used for a variety of purposes, such as data analysis, visualization, or integration with other software systems.
```

Comment:
> as well as any custom fields that have been defined by the community.

Gives the impression that we have per-community custom fields... Should it just say "as well as custom fields"